kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -214,6 +222,10 @@ data:
     # The following settings are "undocumented", we don't recommend uncommenting them unless directed by Microsoft.
     # [agent_settings.telemetry_config]
     #   disable_telemetry = false  # if this is not applied, default value is false
+    
+    # The following setting is for KubernetesMetadata CacheTTL Settings, default is 60s.
+    # [agent_settings.k8s_metadata_config]
+    #   kube_meta_cache_ttl = 60  # if this is not applied, default value is false
```

Comment:
1. rename the field name to have suffix seconds i.e. kube_meta_cache_ttl_seconds
2. fix the comment, "if this is not applied, default value is false" since this is not correct.

Reply:
changed